lib/views/home_content/map_feed/map_feed.dart

```diff
@@ -0,0 +1,71 @@
+import "package:flutter/material.dart";
+import "package:google_maps_flutter/google_maps_flutter.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/services/geolocation_service.dart";
+import "package:proxima/views/home_content/map_feed/maps/nearby_posts_map.dart";
+import "package:proxima/views/sort_option_widgets/feed_sort_option/map_sort_option_chips.dart";
+
+class MapFeed extends ConsumerStatefulWidget {
+  const MapFeed({super.key});
+
+  @override
+  MapFeedState createState() => MapFeedState();
+}
+
+class MapFeedState extends ConsumerState<MapFeed> {
+  //key for testing
+  static const mapScreenKey = Key("map");
+  static const dividerKey = Key("Divider");
+
+  //current position of the user
+  late LatLng _currentPosition;
+
+  //current zoom level of the map
+  static const double _currentZoom = 14;
+
+  //radius of the circle (in km)
+  static const double _radius = 1;
```

Comment:
I think that we should have a 100m radius as it is the radius that we defined for the posts in the overview